subgraph/README.md

```diff
@@ -36,23 +36,61 @@ Are Unlock's contracts are upgradable, we parse the multiple ABIs that are requi
 yarn prepare:abis
 ```
 
+#### Generating Manifest & Building Subgraph
+
+To build any network. This will generate the network configs, prepare the abis and generate the manifest for the subgraph according to the network configuration.
+
+```sh
+# build & generate
+yarn run build <network-name>
+```
+
 ### Deploying The Subgraph
 
-Deploy the latest subgraph code to the graph node.
+Deploy the latest subgraph code to the hosted-service.
 
 ```sh
 export SUBGRAPH_DEPLOY_KEY=<api-key>
 
-# build
-yarn run build <network-name>
-
 # deploy a single network
 yarn run deploy <network-name>
 
-# deploy all networks
+# deploy all available networks to hosted service
 yarn run deploy-all
 ```
 
+Deploy the latest subgraph code to the studio.
+
+```sh
+export SUBGRAPH_STUDIO_DEPLOY_KEY=<api-key>
+
+# deploy a single network
+yarn run deploy <network-name> --studioName=<Name of studio deployment> --label=<Studio version>
+
+```
+
+Direct deployments can be made to the hosted service.
```

Comment:
What is the difference between a "direct deployment" and the above?

Reply:
Direct deployment does not use the `endpointV2` in networks to get the deployment address slug like the other deploy. This allows direct deployment to a endpoint which is passed in as a parameter. If someone would like to fork or do a quick deployment without changing code to a different address, this will come in handy.